bot/services/ChatBot.cs

```diff
@@ -214,6 +214,11 @@ private async Task CompleteGenerationAsync(HttpClient httpClient, string userId,
         switch (chatResponse.Intent)
         {
             case Intent.Unknown:
+                if (status == "No documents found.")

```

Comment:
This is only 1 case. There is also the case where the documents are returned and Generate Answer does not not think any of them answer the question.

Reply:
Is there an example of that I can test with?

Reply:
is there another screen for entering shipping info?

How do I enter a new customer?

I think those.

Reply:
updated per discussion